src/store/project/project-reducer.test.ts

```diff
@@ -28,7 +28,7 @@ describe('project-reducer', () => {
                 id: "2",
                 items: [],
                 data: mockProjectResource({ uuid: "2" }),
-                status: 0
+                status: TreeItemStatus.Initial
```

Comment:
TreeItemStatus is UPPER_CASED in another PR.

Reply:
I'll change it after I merge branch with snakeCase => UPPER_CASE